src/Microsoft.DocAsCode.Build.Engine/TemplateProcessors/Preprocessors/TemplateJintPreprocessor.cs

```diff
@@ -80,11 +78,7 @@ public TemplateJintPreprocessor(IResourceFileReader resourceCollection, Resource
         {
             if (!string.IsNullOrWhiteSpace(scriptResource.Content))
             {
-                _engine = SetupEngine(resourceCollection, scriptResource, context);
-            }
-            else
-            {
-                _engine = null;
+                SetupEngine(resourceCollection, scriptResource, context);
```

Comment:
`_engine = ...`

Reply:
@superyyrrzz is that necessary? it doest seem _engine  is used?

Reply:
You are right... Just notice this.